### PR TITLE
Add details for Friendica

### PR DESCRIPTION
**Pull request by mexon**
I wanted a pretty logo for Friendica.  This also brings in a bunch of other details.

### CODE REVIEW

File: software.json [hunk 1]
```diff
@@ -210,15 +210,15 @@
...
-        "support_url": null,
-        "forum_url": null,
+        "support_url": "https://forum.friendi.ca/profile/helpers",
+        "forum_url": "https://forum.friendi.ca/directory",
         "donate_url": null,
         "matrix_url": null,
-        "logo_source_url": null,
-        "logo_source_version": null
+        "logo_source_url": "https://github.com/friendica/friendica/blob/2024.09-rc/images/friendica-1600.png",
+        "logo_source_version": 2
```
[the remark this answers is not in the document]
Comment (dansup) [reply software.json:221]: Please update the `logo_source_url` with 

```
https://raw.githubusercontent.com/friendica/friendica/refs/heads/2024.09-rc/images/friendica-1600.png
```

(We need a direct link to fetch the image)

**Review by dansup — CHANGES_REQUESTED** [review]
Please update the `logo_source_url` with 

```
https://raw.githubusercontent.com/friendica/friendica/refs/heads/2024.09-rc/images/friendica-1600.png
```

(We need a direct link to fetch the image)

**Comment by mexon** [comment]
> (We need a direct link to fetch the image)

Oh right, of course.  Done.

**Review by dansup — APPROVED** [review]

**Comment by dansup** [comment]
@mexon Done!

https://fedidb.org/software/friendica

**Pull request merged**